cabal-testsuite/PackageTests/Regression/T4154/time.cabal

```diff
@@ -4,7 +4,7 @@ cabal-version:   >=1.8
 build-type:      Custom
 
 custom-setup
-  setup-depends: base, Cabal == 99999
```

Comment:
I think that Cabal-99999 can be deleted now that it doesn't fit the implicit setup constraints.